azurerm/data_source_route_table.go

```diff
@@ -0,0 +1,139 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmRouteTable() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmRouteTableRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": resourceGroupNameSchema(),
```

Comment:
Should this be `resourceGroupNameForDataSourceSchema()`?

Reply:
👍 fixed